### PR TITLE
feat: add SourceIcon

### PR DESCRIPTION
**Pull request by mdeetman**
image
image

### CODE REVIEW

File: src/components/SourceIcon.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import MailOutlineIcon from '@material-ui/icons/MailOutline';
+import WebIcon from '@material-ui/icons/Web';
+
+export default function SourceIcon(props) {
```
Comment (jasonenglish) [inline src/components/SourceIcon.js:5]: love this naming

File: src/components/SourceIcon.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    const SrcIcon = props.source === 'WEB' ? WebIcon : MailOutlineIcon;
+
+    return (
+        <SrcIcon style={{ color: '#B4B8C0' }} />
```
Comment (jasonenglish) [inline src/components/SourceIcon.js:9]: thoughts on simplifying this name to just `Icon`?

File: package.json [hunk 3]
```diff
@@ -45,10 +45,12 @@
   },
   "dependencies": {
     "@material-ui/core": "^4.10.2",
+    "@material-ui/icons": "^4.9.1",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "fontsource-roboto": "^2.1.4"
+    "fontsource-roboto": "^2.1.4",
```
Comment (jasonenglish) [inline package.json:52]: is roboto required for the icons?

Comment (mdeetman) [reply package.json:52]: no. that must have happened when i removed the node modules file and ran npm install again

Comment (mdeetman) [reply package.json:52]: It was already part of the package.json. Maybe from the components added by other people?

Comment (jasonenglish) [reply package.json:52]: it was added in the other repo tho?

Comment (jasonenglish) [reply package.json:52]: if you remove this line, does everything still function? if yes, then we can remove it for now until we need it

File: src/components/SourceIcon.js (hunk 1 above)
Comment (jasonenglish) [inline src/components/SourceIcon.js:5]: i forgot a few things. we can actually change `SourceIcon(props)` to be `SourceIcon({ source })` so we can ref the source variable directly.

we should also specify what values are allowed for `source`, we can do this using Reacts PropTypes: https://reactjs.org/docs/typechecking-with-proptypes.html

i think we'd want to use `PropTypes.oneOf`, which you can find examples of on the above docs

**Pull request merged**